Jenkinsfile.github

```diff
@@ -79,7 +79,7 @@ pipeline {
 
                     axis {
                         name 'DOCKER_TAG'
-                        values '3.11-SLES15.6', '3.11-SLES15.5', '3.10', '3.9', '3.6'
+                        values '3.12'
```

Comment:
Any arm in adding 3.12 without dropping the rest?
We can drop 3.11-SLES15.6, we effectively need one for CSM 1.4 and newer (15-SP3).